### PR TITLE
Breakpoints yaml to sass parser

### PR DESCRIPTION
**Pull request by iberdinsky-skilld**

### CODE REVIEW

**Comment by iberdinsky-skilld** [comment]
Suggestion is to avoid usage of http://breakpoint-sass.com/
We mostly use simple queries. and it needs own query syntax which cannot be parsed easy.

File: gulp-tasks/breakpoints.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+  let scssMap = '';
+
+  scssMap += '// ' + WARNING + '\n';
+
+  scssMap += '$breakpoints-groups: (\n';
+
+  groups.forEach((queries, group) => {
+
+    let groupPostfix;
```
Comment (nguerrier-skilld) [inline gulp-tasks/breakpoints.js]: I'd recommend "groupSuffix" as variable name : 

```suggestion
    let groupSuffix;
```

File: gulp-tasks/breakpoints.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    if (group === themeName) {
+      groupPostfix = '--default';
```
Comment (nguerrier-skilld) [inline gulp-tasks/breakpoints.js]:
```suggestion
      groupSuffix = '--default';
```

File: gulp-tasks/breakpoints.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    }
+    else {
+      groupPostfix = '--' + group.replace('.', '_') + '-group';
```
Comment (nguerrier-skilld) [inline gulp-tasks/breakpoints.js]:
```suggestion
      groupSuffix = '--' + group.replace('.', '_') + '-group';
```

File: gulp-tasks/breakpoints.js [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+    }
+
+    scssMap += '  bp-' + themeName + groupPostfix + ': (\n';
```
Comment (nguerrier-skilld) [inline gulp-tasks/breakpoints.js]:
```suggestion
    scssMap += '  bp-' + themeName + groupSuffix + ': (\n';
```

**Comment by iberdinsky-skilld** [comment]
for tests you can use https://cgit.drupalcode.org/drupal/tree/core/modules/breakpoint/tests test configs. with groups and multipliers

**Pull request merged**